src/ChallengeIntroModal3/stories.js

```diff
@@ -0,0 +1,17 @@
+import {stories, string, className, bool, number, func} from '../duckyStories';
+import ChallengeIntroModal3 from './index';
+
+stories(module, ChallengeIntroModal3, [
+  'https://github.com/DuckyTeam/ducky-web/issues/1707'
+], {
```

Comment:
pass in the image prop: `image: string('https://www.ducky.no/images/landing-image-1.jpg')`